tedana/metrics/kundu_fit.py

```diff
@@ -126,16 +123,17 @@ def dependence_metrics(catd, tsoc, mmix, t2s, tes, ref_img,
     # compute Betas and means over TEs for TE-dependence analysis
     betas = get_coeffs(utils.unmask(catd, mask),
                        mmix,
-                       np.repeat(mask[:, np.newaxis], len(tes), axis=1))
+                       np.repeat(mask[:, np.newaxis], len(tes), axis=1),
```

Comment:
Since len(tes) is fixed here and the adaptive mask isn't used, it seems like betas is being calculated on echoes that might be outside the adaptive mask.

Reply:
`betas` are calculated for all echoes at all voxels, and then only good voxels and good echoes are used later on.